core-services/release-controller/_repos/ocp-4.11-rhel85.repo

```diff
@@ -55,46 +55,10 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-8-advanced-virt]
 name = rhel-8-advanced-virt
-baseurl = https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/advanced-virt/os/
+baseurl = https://cdn.redhat.com/content/eus/rhel8/x86_64/advanced-virt/os/
```

Comment:
I've realized that this is maybe not what we want right now as we have: https://github.com/openshift/os/pull/796.
But I'm not sure. Should we also revert that until we go back to 8.6?

Reply:
Eduardo has a PR to remove RHEL-AV as part of 8.6 - https://github.com/openshift/release/pull/26811